### PR TITLE
Update Actions.srt to reflect new mail binding

### PR DESCRIPTION
**Pull request by vzorglub**
Update Actions.srt to reflect new mail binding

### CODE REVIEW

File: Sphinx/Guides/Actions.rst [hunk 1]
```diff
@@ -170,8 +170,13 @@ Use an Addon/Bundle Action
 
         .. code-block::
 
+            #deprecated use the new Mail binding
```
Comment (openhab-5iver) [inline Sphinx/Guides/Actions.rst]: Let's put the new one on top!

**Comment by vzorglub** [comment]
I changed it inside GitHub
How do I change it again?

**Comment by openhab-5iver** [comment]
Thank you! Similar to the OH repos, I've been requiring sign-offs on commits... https://www.openhab.org/docs/developer/contributing.html#sign-your-work. You can either setup a local repo and use git to amend your commit, or close this PR and submit another (through the web) and include...

    Signed-off-by: Joe Smith <joe.smith@email.com>

... at the bottom of the commit message.

File: Sphinx/Guides/Actions.rst (hunk 1 above)
Comment (openhab-5iver) [reply Sphinx/Guides/Actions.rst]: To do it through the GH web client, go to Files Changed, click the elipsis at the top right, select Edit File and make the change.

Comment (openhab-5iver) [reply Sphinx/Guides/Actions.rst]: Or change it when you close this to add the signoff! :smile:

**Pull request closed without merging**